src/applications/personalization/profile/tests/e2e/direct-deposit/vye/mailingAddress.cypress.spec.js

```diff
@@ -0,0 +1,127 @@
+import { PROFILE_PATHS } from '@@profile/constants';
+import mockPersonalInformation from '@@profile/tests/fixtures/personal-information-success.json';
+import mockServiceHistory from '@@profile/tests/fixtures/service-history-success.json';
+import { UPDATED_USER_MOCK_DATA, mockUser } from './login';
+import { generateFeatureToggles } from '../../../../mocks/endpoints/feature-toggles';
+
+describe('Contact information', () => {
+  beforeEach(() => {
+    cy.login(mockUser);
+    cy.intercept('v0/profile/personal_information', mockPersonalInformation);
+    cy.intercept('v0/profile/service_history', () => mockServiceHistory);
+    cy.intercept('GET', '/vye/v1', {
+      statusCode: 200,
+      body: UPDATED_USER_MOCK_DATA,
+    });
+    cy.intercept(
+      'GET',
+      '/v0/feature_toggles*',
+      generateFeatureToggles({
+        toggleVyeAddressDirectDepositFormsInProfile: true,
+      }),
+    );
+    cy.visit(PROFILE_PATHS.CONTACT_INFORMATION);
+    cy.get('[href="/profile/direct-deposit"]')
+      .last()
+      .click();
+    cy.injectAxe();
+  });
+  const fillForm = () => {
+    cy.get('[href="/profile/direct-deposit"]')
+      .last()
+      .click();
+    cy.get('[id="VYE-mailing-address-button"]').click();
+    cy.get('select[name="root_countryCodeIso3"]')
+      .first()
+      .select('United States');
+    cy.get('input[name="root_addressLine1"]').type('322 26th ave apt 1');
+    cy.get('input[name="root_city"]').type('San Francisco');
+    cy.get('select[name="root_stateCode"]')
+      .last()
+      .select('California');
+    cy.get('input[name="root_zipCode"]').type('94121');
+  };
+
+  it('Should expand the form when Edit button is clicked', () => {
+    cy.injectAxeThenAxeCheck();
+    cy.get('[id="VYE-mailing-address-button"]').click();
+    cy.get(
+      '[class="vads-u-margin-y--2 vads-u-line-height--4 vads-u-font-size--base vads-u-font-family--sans"]',
```

Comment:
careful with these class selectors as they are pretty easy to break when having to change minor styling via classnames.

Reply:
should I merge these changes and fix those later @adamwhitlock1 ?